lib/services/platform-service.ts

```diff
@@ -243,13 +244,19 @@ export class PlatformService implements IPlatformService {
 			emulatorServices.checkAvailability().wait();
 
 			this.buildPlatform(platform).wait();
-
 			var packageFile = this.getLatestApplicationPackageForEmulator(platformData).wait().packageName;
 			this.$logger.out("Using ", packageFile);
 
-			var logFilePath = path.join(platformData.projectRoot, this.$projectData.projectName, "emulator.log");
+			var projectName = options["debug-brk"] ? this.$projectData.projectName + "WithInspector" : this.$projectData.projectName;
+			var logFilePath = path.join(platformData.projectRoot, projectName, "emulator.log");
 
 			emulatorServices.startEmulator(packageFile, { stderrFilePath: logFilePath, stdoutFilePath: logFilePath }).wait();
+
+			if (platform === "ios" && options["debug-brk"]) {
+				var downloadedPackagePath = this.$npm.install("tns-ios", ["g"]).wait();
```

Comment:
This does not respect the semver convention we are trying to follow, because it will install the latest version of the runtime which debugging tools might not be compatible with the current runtime used in the app.